### PR TITLE
Member Profile of Jerens Lensun

### LINKED ISSUE #5835: [Community] Member Profile of Jerens Lensun
**Current Behavior**

@jerensl has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @jerensl as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: https://github.com/jerensl
- Twitter: https://x.com/jerensl22
- LinkedIn: https://www.linkedin.com/in/jerensl
- Layer5 Cloud: https://meshery.layer5.io/user/2e246a4a-11e3-4b99-8c79-e42f8565e4be
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Spoorthy1423**
**Description**
#5835 [Community] Created member profile of Jerens Lensun

This PR fixes #5835 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ Yes] Yes, I signed my commits.

### CODE REVIEW

**Comment by hargunkaur286** [comment]
@Spoorthy1423 Thank you for working on this pr. A few points to note: For the "position" placeholder, we give the position that the member has in the community i.e. of a "contributor", and the layer5 profile's link is given in the issue description, please refer other files to see how will it be incorporated.

File: src/collections/members/jeres-lensun/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+twitter: jerensl22
+linkedin: jerensl
+layer5: # (This detail wasn't available in the screenshot)
+location: Indonesia
+bio: |
+  "I am a Software Engineer currently contributing to OSS and working with my personal website. 
+  I regularly write articles on my blog and enjoy talking about Tech and Football. 
+  Feel free to connect with me on various social platforms like LinkedIn, Twitter, and Instagram."
+badges: 
+  - ui-ux
```
Comment (sudhanshutech) [inline src/collections/members/jeres-lensun/index.mdx:15]: remove these two badge and meshery badge please

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/collections/members/jeres-lensun/index.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Jerens Lensun
+position: Software Engineer
+image_path: ./jerens-lensun.jpeg
```
Comment (vishalvivekm) [inline src/collections/members/jeres-lensun/index.mdx:4]:
```suggestion
image_path: ./JerensProfilePic.jpg
```

File: src/collections/members/jeres-lensun/index.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+bio: |
+  "I am a Software Engineer currently contributing to OSS and working with my personal website. 
+  I regularly write articles on my blog and enjoy talking about Tech and Football. 
+  Feel free to connect with me on various social platforms like LinkedIn, Twitter, and Instagram."
+blurb: "Bring impact through code."
+status: Active
+maintainer: no
+meshmate: no
+emeritus: no
+community_manager: no
```
Comment (vishalvivekm) [inline src/collections/members/jeres-lensun/index.mdx:18]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit 629f5ee86b1fbc699403eeab555f37fff7260273 at: https://66ce4757d1e9df8b9080359e--layer5.netlify.app

**Comment by Spoorthy1423** [comment]
@vishalvivekm  i passed all the checks and if anything more needed let me know.

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**